thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/anomaly/merge/AnomalyTimeBasedSummarizer.java

```diff
@@ -117,13 +113,7 @@ public int compare(RawAnomalyResultDTO o1, RawAnomalyResultDTO o2) {
     return mergedAnomalies;
   }
 
-
-  private static void populateMergedResult(MergedAnomalyResultDTO mergedAnomaly,
-      RawAnomalyResultDTO currentResult) {
-    if (!mergedAnomaly.getAnomalyResults().contains(currentResult)) {
-      mergedAnomaly.getAnomalyResults().add(currentResult);
-      currentResult.setMerged(true);
-    }
+  private static void populateMergedResult(MergedAnomalyResultDTO mergedAnomaly, RawAnomalyResultDTO currentResult) {
```

Comment:
This method is used to initialize a new merged anomaly by copying the information of an existing raw anomaly. So, the removed code wasn't doing anything because the mergedAnomaly is always an empty anomaly.